arches/app/media/js/viewmodels/card-component.js

```diff
@@ -125,7 +125,29 @@ define([
         };
 
         this.beforeMove = function(e) {
```

Comment:
I don't get a banner / error when trying to reorder an entire card (makes sense, that's a separate endpoint)